fe/fe-core/src/main/java/org/apache/doris/load/loadv2/LoadingTaskPlanner.java

```diff
@@ -182,6 +182,13 @@ public String getTimezone() {
             }
         }
 
+        // If this is a dynamic partitioned table, it will take some time to create the partition after the
+        // table is created, a exception needs to be thrown here
+        if (partitionIds.isEmpty()) {
+            throw new LoadException("data cannot be inserted into table with emtpy partition. " +
```

Comment:
It is best to guide users in the prompt to wait for a while because of dynamic partitioning

Reply:
I agree. I think we can change the error msg like:
```
data cannot be inserted into table with emtpy partition... Use `SHOW PARTITIONS FROM tbl` to see the currenty partitions of this table. 
```

And also change the msg in error code `ERR_EMPTY_PARTITION_IN_TABLE`

Reply:
ok. I will change the error msg.